# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/trunk/webrtc/webrtc.gni
# Commit: f9a9da6129c0
# Full Hash: f9a9da6129c01b5cefdc2ba2871d5bfe9fd0f0ab
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2019-04-17 21:47:29
# Regressor Bug: 1496359
# File Overlap Count: 1
# Description:
#   Bug 1496359 - Conflict resolutions for Pipewire support patch; r=ng
#   
#   Depends on D27368
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D27369
# ==============================================================================

diff -r 3f0981f37c1c -r f9a9da6129c0 media/webrtc/trunk/webrtc/webrtc.gni
--- a/media/webrtc/trunk/webrtc/webrtc.gni	Mon Apr 15 17:26:27 2019 +0000
+++ b/media/webrtc/trunk/webrtc/webrtc.gni	Mon Apr 15 17:31:02 2019 +0000
@@ -70,6 +70,9 @@
   # Set this to true to enable BWE test logging.
   rtc_enable_bwe_test_logging = false
 
+  # Set this to use PipeWire on the Wayland display server.
+  rtc_use_pipewire = false
+
   rtc_enable_android_opensl = false
 
   # Link-Time Optimizations.
@@ -196,7 +199,8 @@
 rtc_opus_dir = "//third_party/opus"
 
 # Desktop capturer is supported only on Windows, OSX and Linux.
-rtc_desktop_capture_supported = is_win || is_mac || (is_linux && use_x11) || is_bsd
+rtc_desktop_capture_supported =
+    is_win || is_mac || (is_linux && use_x11) || is_bsd || rtc_use_pipewire
 
 ###############################################################################
 # Templates
